packages/preview/src/browser/markdown/markdown-preview-handler.ts

```diff
@@ -55,6 +55,7 @@ export class MarkdownPreviewHandler implements PreviewHandler {
         const sanitizedContent = DOMPurify.sanitize(renderedContent);
         const contentElement = document.createElement('div');
         contentElement.classList.add(this.contentClass);
+        // eslint-disable-next-line no-unsanitized/property
         contentElement.innerHTML = sanitizedContent;
```

Comment:
Minor: We don't need to disable ESlint here if we remove the `sanitizedContent` step.
```suggestion
        const contentElement = document.createElement('div');
        contentElement.classList.add(this.contentClass);
        contentElement.innerHTML = DOMPurify.sanitize(renderedContent);
```